node_modules/gh-apps/lib/api.js

```diff
@@ -294,3 +295,56 @@ var updateAppAdmins = module.exports.updateAppAdmins = function(ctx, id, adminUp
         });
     });
 };
+
+/* Terms and Conditions */
+
+/**
+ * Get the Terms and Conditions for an application
+ *
+ * @param  {Context}        ctx                             Standard context containing the current user and the current app
+ * @param  {Number}         id                              The id of the application for which to retrieve the terms and conditions
+ * @param  {Function}       callback                        Standard callback function
+ * @param  {Object}         callback.err                    An error object, if any
+ * @param  {Object}         callback.termsAndConditions     The Terms and Conditions for the application
+ */
+var getTermsAndConditions = module.exports.getTermsAndConditions = function(ctx, id, callback) {
+    // Ensure that the app id is a valid number
+    id = GrasshopperUtil.getNumberParam(id);
+
+    // Validation
+    var validator = new Validator();
+    validator.check(id, {'code': 400, 'msg': 'A valid app id must be provided'}).isInt();
+    if (validator.hasErrors()) {
+        return callback(validator.getFirstError());
+    }
+
+    // Verify that the provided application exists
+    getApp(ctx, id, function(err, app) {
+        if (err) {
+            return callback(err);
+        }
+
+        var termsAndConditions = _getTermsAndConditions(app.id);
+        return callback(null, termsAndConditions);
+    });
+};
+
+/**
+ * Get the Terms and Conditions for an application.
+ *
+ * This method performs no validation on the given application id and should only be used when
+ * you're sure the application exists. It's only exported because some dependencies shouldn't incur
+ * the overhead of an extra DB query that happens through `getTermsAndConditions`
+ *
+ * @param  {Number}         id                              The id of the application for which to retrieve the terms and conditions
+ * @return {Object}         termsAndConditions              The Terms and Conditions for the application
+ * @throws {Error}                                          Error when the application does not exist
+ * @api private
+ */
+var _getTermsAndConditions = module.exports._getTermsAndConditions = function(id) {
```

Comment:
I should add that then we can make it a legit public method and not a `_`-prefixed that is exported.